frontend/src/pages/project/main/Main.module.scss

```diff
@@ -0,0 +1,112 @@
+@import "styles/spacings";
+@import "styles/colors";
+@import "styles/typography";
+
+.root {
+  display: grid;
+  height: 58vh;
+  column-gap: 15px;
+  row-gap: 20px;
+  grid-template-columns: repeat(2, 1fr);
+
+  .card {
+    height: 100%;
+    border: 0.5px solid #70707048;
+    border-radius: 0px;
+
+    .content {
+      display: flex;
+      flex-direction: column;
+      margin: $spacing-xl;
+      margin-top: $spacing-xl * 1.5;
+
+      .title {
+        display: flex;
+        font-size: $font-size-h2;
+        font-weight: 600;
+
+        .counter {
+          margin-left: $spacing-l;
+          margin-top: 2px;
+          background-color: #5c6474;
+          border-radius: 9px;
+          font-size: $font-size-xsmall;
+          padding: 0px 12px 0px 12px;
+        }
+      }
+
+      .items {
```

Comment:
Nit: Maybe add some prefix to this class name, like `itemsWrapper` or `itemsContainer`?